packages/plugin-constraints/sources/commands/constraints.ts

```diff
@@ -35,51 +35,62 @@ export default class ConstraintsCheckCommand extends BaseCommand {
 
   @Command.Path(`constraints`)
   async execute() {
-    for (let t = 0; t < 10; ++t) {
-      const configuration = await Configuration.find(this.context.cwd, this.context.plugins);
-      const {project} = await Project.find(configuration, this.context.cwd);
-      const constraints = await Constraints.find(project);
+    const configuration = await Configuration.find(this.context.cwd, this.context.plugins);
+    const {project} = await Project.find(configuration, this.context.cwd);
+    const constraints = await Constraints.find(project);
 
-      const toSave = new Set<Workspace>();
+    const report = await StreamReport.start({
+      configuration,
+      stdout: this.context.stdout,
+    }, async report => {
+      let allSaves = new Set<Workspace>();
+      const errors = new Set<[MessageName, string]>();
```

Comment:
Better use an array, since the array literals will never match the existing items.

Reply:
yeah initially I used an array but I decised to move to a Set to align to existing codebase style. 
Will replace with an array.